src/controllers/account.js

```diff
@@ -0,0 +1,39 @@
+const createUserModel = require("../models/account");
+const { DataTypes } = require("sequelize");
+
+const { db } = require("../databaseSetup");
+
+const Account = createUserModel(db, DataTypes);
+
+const blockAccount = async (req, res) => {
+  const { userId } = req.params;
+  console.log(userId);
+
+  await Account.update(
+    { isBlocked: 1 },
+    {
+      where: {
+        userId: userId,
```

Comment:
Simplify to single userId

Fix same issue in other places

Reply:
I agree